Tests/MongoKittenTests/CRUDTests.swift

```diff
@@ -1,404 +1,168 @@
-////                    let pipeline = try Pipeline().match(query).count(writingInto: "pets")
-////
-////                    return pets.aggregate(pipeline)
-////                } catch {
-////                    return connection.eventLoop.makeFailedFuture(error)
-////                }
-////            }
-////        }.wait()
-////
-////        XCTAssertEqual(pets, 4)
-////    }
-//}
-//
-////extension MongoCollection {
-////    func addPet(named name: String, owner: String) -> EventLoopFuture<Void> {
-////        return self.insert([
-////            "_id": self.objectIdGenerator.generate(),
-////            "name": name,
-////            "owner": owner
-////        ]).map { _ in }
-////    }
-////}
-////
-////extension EventLoopFuture where T == Cursor<Document> {
-////    func testRange(startingAt start: Int64 = 10, count: Int64 = 10) -> EventLoopFuture<Void> {
-////        return self.flatMap { cursor in
-////            var n: Int64 = start
-////
-////            return cursor.forEach { document in
-////                XCTAssertEqual(document["n"] as? Int64, n)
-////                n += 1
-////            }.map {
-////                XCTAssertEqual(n, start + count)
-////            }
-////        }
-////    }
-////}
+import NIO
+import MongoKitten
+import XCTest
+import MongoCore
+
+let dbName = "KittenTest"
+
+class CrudTests : XCTestCase {
+    struct DummyAccount: Codable {
+        static let collectionName = "DummyAccounts"
+        
+        let _id: ObjectId
+        var name: String
+        var password: String
+        var age: Int
+        
+        init(name: String, password: String, age: Int) {
+          self._id = ObjectId()
+          self.name = name
+          self.password = password
+          self.age = age
+        }
+    }
+    
+    let settings = try! ConnectionSettings("mongodb+srv://AMTester:Autimatisering1@amtestcluster.n5lfc.mongodb.net/\(dbName)?retryWrites=true&w=majority")
+    
+    //let settings = ConnectionSettings(
+    //  authentication: .auto(username: "admin", password: "Autimatisering1"),
+    //  authenticationSource: "admin",
+    //  hosts: [
+    //      .init(hostname: "localhost", port: 27017)
+    //  ],
+    //  targetDatabase: dbName,
+    //  applicationName: "Test MK6"
+    //)
+    
+    var mongo: MongoDatabase!
+    var concern: WriteConcern!
+    
+    override func setUpWithError() throws {
+        mongo = try MongoDatabase.synchronousConnect(settings: settings)
+        concern = WriteConcern()
+        concern.acknowledgement = .majority
+    }
+    
+    override func tearDownWithError() throws {
+        try mongo.drop().wait()
+    }
+    
+    func printDummyAccounts () throws {
+        let schema = mongo[DummyAccount.collectionName]
+        try schema.find().forEach{ dummy in
+            print("\(dummy["_id"]!): Name - \(dummy["name"]!) | password - \(dummy["password"]!) | age - \(dummy["age"]!)")
```

Comment:
Force unwraps. Also, before releasing we'll need to get rid of any print statements. Replace them with assertions.